src/test/java/Homework19.java

```diff
@@ -0,0 +1,63 @@
+import com.beust.jcommander.Parameter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class Homework19 extends BaseTest{
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        loginPage.login();
+        homePage.selectPlaylist();
+        homePage.clickDeletePlaylistBtn();
+        homePage.getPlaylistDeletedMessage();
+
+       // Assert.assertEquals(getPlaylistDeletedMessage(),deletedPlaylistSuccessMessage);
+        Assert.assertTrue(homePage.getPlaylistDeletedMessage().
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it.

![image](https://github.com/Test-Pro-Bootcamp/A55-QA-Automation-Framework/assets/104761482/65fc1174-b39d-49f0-bb53-3d44e61ed740)

HINT

**Step 1**

**HomePage**

![image](https://github.com/Test-Pro-Bootcamp/A55-QA-Automation-Framework/assets/104761482/45f7c108-74b7-4dcb-8acb-d78d33a53c0f)

**Step 2**

![image](https://github.com/Test-Pro-Bootcamp/A55-QA-Automation-Framework/assets/104761482/d4e4f719-c4f6-4c1d-8c84-32e93cdfe097)